ui/js/controllers/perf/dashboard.js

```diff
@@ -1,4 +1,3 @@
-import _ from 'lodash';
 import chunk from 'lodash/chunk';
```

Comment:
I'd like to request that we take the same approach as we did with chunk here. Import the method at toplevel, then use it directly inside the code without the lodash prefix. The `for (const x in foo)` approach is IMO less clear.

So change this to:

```js
import chunk from 'lodash/chunk';
import forIn from 'lodash/forIn';
```

Reply:
The lodash chunk instance definitely made sense to leave as lodash (but switch to a direct import), since the native JS equivalent was many many times longer.

However for `forIn` it's not really any more verbose, so I'm not entirely convinced we should keep using it. Another factor is that the native "for in" approach is going to be something developers become more and more familiar with over time vs third party libraries, even if it feels "new" now.

Reply:
The `for in` pattern took several lines more and a bunch of object dereferencing inside a file that was *already* extremely hard to read and properly understand.

Both the files touched should be rewritten or removed (dashboard.js isn't even accessible via the ui!). For now can we please just leave this be? I really don't want to spend any more time on this code than absolutely necessary to keep it working.

Reply:
Yeah if they are being rewritten at some point that sounds fine :-)